src/db.rs

```diff
@@ -137,6 +139,8 @@ pub async fn list_available_hosts(db: &Client, cutoff: u64) -> Result<Vec<HostSt
                 "wanIp": "$wanIp",
                 "timestamp":"$timestamp",
                 "hposAppList": "$hposAppList",
+                "channelVersion": "$channelVersion",
+                "hposVersion": "$hposVersion",
```

Comment:
Is this correct @peeech? this seems to fit the form of the adjacent elements, but your diff said to use 
```
{"$first": "$hposVersion"}
```

Reply:
Obviously you're right :-)